WorkloadManagementSystem/Executor/Base/OptimizerExecutor.py

```diff
@@ -5,7 +5,6 @@
 __RCSID__ = "$Id$"
 
 import threading
-import datetime
```

Comment:
Did you verify that this is not needed? The comment below this line refers to this import...
See also: https://github.com/DIRACGrid/DIRAC/pull/3748#discussion_r200631554
If it is not needed that comment should be dropped

Reply:
Correct, fixed.